website/docs/configuration/resources.html.md

```diff
@@ -299,6 +300,52 @@ intended. The practice of generating multiple instances from lists should
 be used sparingly, and with due care given to what will happen if the list is
 changed later.
 
+### `for_each`: Multiple Resource Instances Defined By a Map
```

Comment:
`for_each` should be able to work with a list or a map (even if it doesn't now, it should before we merge this)

Reply:
Will discuss this in chat :)

Reply:
I hereby revoke my comment! MAPS!!!